helm-app-operator/pkg/helm/controller/reconcile.go

```diff
@@ -54,52 +53,60 @@ func (r HelmOperatorReconciler) Reconcile(request reconcile.Request) (reconcile.
 	o.SetGroupVersionKind(r.GVK)
 	o.SetNamespace(request.Namespace)
 	o.SetName(request.Name)
-	logrus.Debugf("Processing %s", util.ResourceString(o))
+	log := log.WithValues(
+		"namespace", o.GetNamespace(),
+		"name", o.GetName(),
+		"apiVersion", o.GetAPIVersion(),
+		"kind", o.GetKind(),
+	)
+	log.V(1).Info("Reconciling")
 
 	err := r.Client.Get(context.TODO(), request.NamespacedName, o)
 	if apierrors.IsNotFound(err) {
 		return reconcile.Result{}, nil
 	}
 	if err != nil {
-		logrus.Errorf("failed to lookup %s: %s", util.ResourceString(o), err)
+		log.Error(err, "failed to lookup resource")
 		return reconcile.Result{}, err
 	}
 
+	manager := r.ManagerFactory.NewManager(o)
+	status := types.StatusFor(o)
+	log = log.WithValues("release", manager.ReleaseName())
+
 	deleted := o.GetDeletionTimestamp() != nil
 	pendingFinalizers := o.GetFinalizers()
 	if !deleted && !contains(pendingFinalizers, finalizer) {
-		logrus.Debugf("Adding finalizer \"%s\" to %s", finalizer, util.ResourceString(o))
+		log.V(1).Info("Adding finalizer", "finalizer", finalizer)
 		finalizers := append(pendingFinalizers, finalizer)
 		o.SetFinalizers(finalizers)
 		err := r.Client.Update(context.TODO(), o)
 		return reconcile.Result{}, err
 	}
 
-	manager := r.ManagerFactory.NewManager(o)
-	status := types.StatusFor(o)
-	releaseName := manager.ReleaseName()
```

Comment:
We no longer need to log the release name?

Reply:
My bad just saw line 75

Reply:
That happens here now:
https://github.com/operator-framework/helm-app-operator-kit/pull/74/files#diff-8b65fa47acae82766fcb9b9647c9ca6eR75